### ISSUE TITLE
dotnet add package should add PackageReference with Version attribute

### ISSUE THREAD

**Issue by DustinCampbell** [dotnet/cli#5386]
## Steps to reproduce
```
dotnet new
dotnet add package --version 4.3.0 System.ValueTuple
```

## Expected  behavior
```XML
<Project Sdk="Microsoft.NET.Sdk">
  <PropertyGroup>
    <OutputType>Exe</OutputType>
    <TargetFramework>netcoreapp1.0</TargetFramework>
  </PropertyGroup>
  <ItemGroup>
    <PackageReference Include="System.ValueTuple" Version="4.3.0" />
  </ItemGroup>
</Project>
```

## Actual behavior
```XML
<Project Sdk="Microsoft.NET.Sdk">
  <PropertyGroup>
    <OutputType>Exe</OutputType>
    <TargetFramework>netcoreapp1.0</TargetFramework>
  </PropertyGroup>
  <ItemGroup>
    <PackageReference Include="System.ValueTuple">
      <Version>4.3.0</Version>
    </PackageReference>
  </ItemGroup>
</Project>
```

## Environment data
`dotnet --info` output:

```
.NET Command Line Tools (1.0.0-rc4-004530)

Product Information:
 Version:            1.0.0-rc4-004530
 Commit SHA-1 hash:  4f0b7bd58c

Runtime Environment:
 OS Name:     Windows
 OS Version:  10.0.15007
 OS Platform: Windows
 RID:         win10-x64
 Base Path:   C:\Program Files\dotnet\sdk\1.0.0-rc4-004530
```

**Comment by piotrpMSFT** [comment]
@rohit21agrawal can you fix this for RTM?

@rrelyea FYI

**Comment by rohit21agrawal** [comment]
tagging @mishra14  who wrote the original feature

**Comment by piotrpMSFT** [comment]
@rohit21agrawal thanks!

**Comment by mishra14** [comment]
@piotrpMSFT @DustinCampbell This was blocked by https://github.com/Microsoft/msbuild/issues/1393 as [commented](https://github.com/NuGet/NuGet.Client/blob/dev/src/NuGet.Core/NuGet.CommandLine.XPlat/Utility/MSBuildAPIUtility.cs#L230)  in NuGet code.

**Comment by rainersigwald** [comment]
@mishra14 It shouldn't be blocked by that--the MSBuild issue would just provide some APIs to make it easier. Should just be a matter of taking the metadata you're already adding and flipping it to be an attribute.

**Comment by mishra14** [comment]
@piotrpMSFT : Spoke to @rrelyea, we are holding off till we get  ship room approval.

**Comment by mishra14** [comment]
NuGet bug: https://github.com/NuGet/Home/issues/4325

**Comment by piotrpMSFT** [comment]
Closing in favor of NuGet/Home#4325
